server_cmds.go

```diff
@@ -15,129 +15,149 @@ type ServerMethods struct {
 	MachineID                              string  `ms:"virtualserver_machine_id"`
-	MaxClients                             int     `ms:"virtualserver_maxclients"`
-	MinClientsInChannelBeforeForcedSilence int     `ms:"virtualserver_min_clients_in_channel_before_forced_silence"`
-	NeededIdentitySecurityLevel            int     `ms:"virtualserver_needed_identity_security_level"`
-	Port                                   int     `ms:"virtualserver_port"`
-	QueryClientsOnline                     int     `ms:"virtualserver_queryclientsonline"`
-	Uptime                                 int     `ms:"virtualserver_uptime"` // TODO(steve): use time.Duration
-	AskForPrivilegeKey                     bool    `ms:"virtualserver_ask_for_privilegekey"`
-	AutoStart                              bool    `ms:"virtualserver_autostart"`
-	FlagPassword                           bool    `ms:"virtualserver_flag_password"`
-	LogChannel                             bool    `ms:"virtualserver_log_channel"`
-	LogClient                              bool    `ms:"virtualserver_log_client"`
-	LogFileTransfer                        bool    `ms:"virtualserver_log_filetransfer"`
-	LogPermissions                         bool    `ms:"virtualserver_log_permissions"`
-	LogQuery                               bool    `ms:"virtualserver_log_client"`
-	LogServer                              bool    `ms:"virtualserver_log_server"`
-	WebListEnabled                         bool    `ms:"virtualserver_web_list_enabled"`
-	PrioritySpeakerDimmModificator         float32 `ms:"virtualserver_priority_speaker_dimm_modificator"`
-	BandwidthReceivedLastMinuteTotal       int     `ms:"virtualserver_bandwidth_received_last_minute_total"`
-	BandwidthReceivedLastSecondTotal       int     `ms:"virtualserver_bandwidth_received_last_second_total"`
-	BandwidthSentLastMinuteTotal           int     `ms:"virtualserver_bandwidth_sent_last_minute_total"`
-	BandwidthSentLastSecondTotal           int     `ms:"virtualserver_bandwidth_sent_last_second_total"`
-	ChannelTempDeleteDelayDefault          int     `ms:"virtualserver_channel_temp_delete_delay_default"`
-	HostBannerMode                         int     `ms:"virtualserver_hostbanner_mode"`
-	IconID                                 int     `ms:"virtualserver_icon_id"`
-	MinAndroidVersion                      int     `ms:"virtualserver_min_android_version"`
-	MinClientVersion                       int     `ms:"virtualserver_min_client_version"`
-	MiniOSVersion                          int     `ms:"virtualserver_min_ios_version"`
-	ReservedSlots                          int     `ms:"virtualserver_reserved_slots"`
-	TotalPing                              float32 `ms:"virtualserver_total_ping"`
+	NamePhonetic                           string  `ms:"virtualserver_name_phonetic"`
+	IP                                     string  `ms:"virtualserver_ip"`
+	ServerNickname                         string  `ms:"virtualserver_nickname"`
```

Comment:
```suggestion
	Nickname                         string  `ms:"virtualserver_nickname"`
```
For naming consistency

Reply:
I would rather not do that because the the nickname is actually called ServerNickname.
Additionally this could cause confusion to the "client" Nickname and "server" Nickname.